src/test/java/org/springframework/data/couchbase/core/CouchbaseTemplateKeyValueIntegrationTests.java

```diff
@@ -74,6 +74,12 @@ void upsertAndFindById() {
 		User modified = couchbaseTemplate.upsertById(User.class).one(user);
 		assertEquals(user, modified);
 
+		modified = couchbaseTemplate.replaceById(User.class).one(user);
+		assertEquals(user, modified);
```

Comment:
On a related note, Spring Data has switched to AssertJ which makes assertions a bit more convenient.